### PR TITLE
Merge all Galley integration tests into a single test suite.

### PR DESCRIPTION
**Pull request by ozevren**
+ Move tests under tests/integration/galley to one folder up, into a common suite.
+ Hoist label & environment requirements from each individual test main, to each individual test, as needed.

### CODE REVIEW

File: tests/integration/galley/dialout_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+`
+
+func TestDialout_Basic(t *testing.T) {
+	// TODO: Limit to Native environment until the Kubernetes environment is supported in the Galley
```
Comment (nmittler) [inline tests/integration/galley/dialout_test.go]: Can any of these tests be run in parallel? If they can, let's mark them so ... if we decide to make separate targets for each suite, we can take advantage of parallelism.

Comment (ozevren) [reply tests/integration/galley/dialout_test.go]: Possibly, but unlikely. For the native case, I'd assume there would be port clashes. The only K8s one is validation, and that should be able to run in parallel, but there is no other K8s test active at the moment. So marking them parallel at this point won't save much.

I'd prefer to that as a separate work, after making all of these environment agnostic first. I can do the parallelization changes as part of that.

**Comment by ozevren** [comment]
/test e2e-mixer-no_auth
--

File: README.md [hunk 2]
```diff
@@ -106,3 +106,8 @@ milestone cannot be considered achieved if the issue isn't resolved.
 
 We don't annotate issues with Releases; Milestones are used instead. We don't use GitHub projects at all, that
 support is disabled for our organization.
+
+
+
+
+Martin made me do this.
```
Comment (howardjohn) [inline README.md]: ?

**Comment by ozevren** [comment]
D'oh. Accidental change when we were testing something.  I'll revert.

*From: *John Howard <notifications@github.com>
*Date: *Fri, May 3, 2019 at 9:30 AM
*To: *istio/istio
*Cc: *Ozben Evren, Mention

*@howardjohn* commented on this pull request.

**Pull request merged**